using_openshift/port_forwarding.adoc

```diff
@@ -0,0 +1,86 @@
+= Port Forwarding
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift takes advantage of a feature built into Kubernetes to support port
+forwarding to pods. This is implemented using HTTP along with a multiplexed
+streaming protocol such as link:http://www.chromium.org/spdy[SPDY] or
+link:https://http2.github.io/[HTTP/2].
+
```

Comment:
@adellape This paragraph is similar to the one above (from the using_openshift/executing_remote_commands.adoc file), the difference being SPDY and HTTP/2 are in bold, but here they're not. Should they be bold?